scripts/md2rst.py

```diff
@@ -29,5 +29,5 @@ def download_pandoc():
 def md2rst(text):
     download_pandoc()
     output = pypandoc.convert_text(text, 'rst', format='md',
-     filters=[os.path.join('filters', 'yml_filter.py')])
+     filters=[os.path.join(os.getcwd(),'filters', 'yml_filter.py')])
```

Comment:
@pri22296 kindly check this . here is the generated [doc](https://sch00lb0y.github.io/tempCheck/) .
my travis [log](https://travis-ci.org/sch00lb0y/tempCheck/jobs/238782122/config). check travis config and here is the @imujjwal96 [repo](https://github.com/imujjwal96/mydoc/blob/master/scripts/md2rst.py)